CREDITS.txt

```diff
@@ -0,0 +1 @@
+Michele Sciabarra <michele@sciabarra.com>
```

Comment:
FWIW I don't believe we'd one credit files like this anywhere else.
I think this is the only credit file I'm aware of although I haven't checked all other repos. https://github.com/apache/incubator-openwhisk/blob/master/CREDITS.txt